core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooQueueLock.java

```diff
@@ -111,4 +115,50 @@ public void removeEntry(long entry) {
       throw new RuntimeException(ex);
     }
   }
+
+  public static List<String> validateAndSortChildrenByLockPrefix(String path,
+      List<String> children) {
+    log.trace("validating and sorting children at path {}", path);
+    List<String> validChildren = new ArrayList<>();
+    if (children == null || children.isEmpty()) {
+      return validChildren;
+    }
+    children.forEach(c -> {
+      log.trace("Validating {}", c);
+      if (c.startsWith(PREFIX)) {
+        int idx = c.indexOf('#');
+        String sequenceNum = c.substring(idx + 1);
+        if (sequenceNum.length() == 10) {
+          try {
+            log.trace("Testing number format of {}", sequenceNum);
+            Integer.parseInt(sequenceNum);
+            validChildren.add(c);
+          } catch (NumberFormatException e) {
+            log.warn("Child found with invalid sequence format: {} (not a number)", c);
```

Comment:
It may be more descriptive if, instead of Child... if it was FateLock (or flock) or something that is more descriptive than generic child - this applies in multiple places.

Reply:
I made some log changes in my latest commit

Reply:
@EdColeman Thoughts on my recent log changes? Is that more descriptive and do you think I should make those same log changes in the validation function inside `ZooLock`? If you approve then I will get this merged in today.

Reply:
I'll look at zoo lock.  That should not hold up this request.  I think this can be merged unless others have issues.